### ISSUE TITLE
Documenting Intel and AMD services used for attestation

### ISSUE THREAD

**Issue by bpradipt** [confidential-containers/trustee#359]
Is there a central location documenting the Intel and AMD services used for attestation?
This info helps to plan for actual deployment w.r.t urls to add to firewall allow list, service SLAs etc.

cc @fitzthum @Xynnn007

**Comment by mythi** [comment]
> Is there a central location documenting the Intel and AMD services used for attestation?

Is the question about where in the CoCo repositories we have this or are you looking for the vendor specific pointers?

**Comment by bpradipt** [comment]
@mythi sorry I should have been clearer. I am looking for vendor specific pointers which can be added to the docs so that it's easier to refer for deployment planning activities. 

Specifically the following details:
- url:port (AMD Key Distribution Server,  Intel Trusted Authority etc..)
- SLA of the service 
- Certificates 
- Reference/Doc  link  (eg https://www.intel.com/content/www/us/en/security/trust-authority.html etc)


May be we can use this issue to list down the relevant details and I can consolidate these and raise a doc PR.

**Comment by fitzthum** [comment]
For AMD no connections are needed from Trustee to external services.

(This is partly because we require the extended cert chain to be provisioned on the host side)

**Comment by fitzthum** [comment]
> Are you referring to SEV specifically or SEV-SNP as well?

Trustee doesn't support SEV, so I am just referring to SNP although the same happens to be true for SEV support in simple-kbs. I think the situation might be a bit different for the Azure SNP verifier.

> I thought Trustee still need to reach out to https://kdsintf.amd.com/

The SNP verifier expects the cert chain to travel with the attestation report (as part of the so-called extended report). This is what you would get from the kds, but basically the host is expected to get this ahead of time. We might add a fallback in the verifier that would contact the KDS, but that is not implemented yet.

**Comment by Xynnn007** [comment]
> The SNP verifier expects the cert chain to travel with the attestation report (as part of the so-called extended report). This is what you would get from the kds, but basically the host is expected to get this ahead of time. We might add a fallback in the verifier that would contact the KDS, but that is not implemented yet.

Do we need to add an issue to record this?

**Comment by fitzthum** [comment]
> Do we need to add an issue to record this?

Good idea. https://github.com/confidential-containers/trustee/issues/363
